selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestSshServiceClient.java

```diff
@@ -14,48 +14,95 @@
 
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
+import com.google.inject.name.Named;
 import java.util.List;
+import org.eclipse.che.api.core.ConflictException;
+import org.eclipse.che.api.core.NotFoundException;
 import org.eclipse.che.api.core.rest.HttpJsonRequestFactory;
 import org.eclipse.che.api.core.rest.HttpJsonResponse;
 import org.eclipse.che.api.ssh.shared.dto.GenerateSshPairRequest;
 import org.eclipse.che.api.ssh.shared.dto.SshPairDto;
 import org.eclipse.che.selenium.core.provider.TestApiEndpointUrlProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /** @author Musienko Maxim */
 @Singleton
 public class TestSshServiceClient {
+  private static final Logger LOG = LoggerFactory.getLogger(TestSshServiceClient.class);
   private static final String MACHINE_SERVICE = "machine";
   private static final String VCS_SERVICE = "vcs";
+
+  public static final String GITHUB_COM = "github.com";
+
   private final String apiEndpoint;
   private final HttpJsonRequestFactory requestFactory;
+  private final TestGitHubServiceClient testGitHubServiceClient;
+  private final String gitHubUsername;
+  private final String gitHubPassword;
 
   @Inject
   public TestSshServiceClient(
-      TestApiEndpointUrlProvider apiEndpointProvider, HttpJsonRequestFactory requestFactory) {
+      TestApiEndpointUrlProvider apiEndpointProvider,
+      HttpJsonRequestFactory requestFactory,
+      TestGitHubServiceClient testGitHubServiceClient,
```

Comment:
I would have a dedicated service to upload VCS key on github and wouldn't mix ssh and github service

Reply:
Did you mean test service client? Class like the follow:
```
TestGitHubKeyUploader
=========================
TestGitHubServiceClient testGitHubServiceClient
TestSshServiceClient testSshServiceClient
--------------------------------------------
void updateGithubKey()
```

Reply:
Something like that.

Reply:
okay